app/main.py

```
@@ -3,31 +3,71 @@
+        if TORTOISE_ORM:
+            await Tortoise.init(config=TORTOISE_ORM)
+            logger.info("Database initialized")
+
+        worker_service = QueueConsumer(
+            queue=supabase_queue,
+            workers=getattr(settings, 'CONSUMER_WORKERS', 4),
+            poll_interval=getattr(settings, 'CONSUMER_POLL_INTERVAL', 2.0),
+            max_processing_time=getattr(settings, 'CONSUMER_MAX_PROCESSING_TIME', 1800)
+        )
+
+        worker_task = asyncio.create_task(worker_service.start())
```

Ruff (0.13.1)

Local variable `worker_task` is assigned to but never used

Remove assignment to unused variable `worker_task`

(F841)